deployment/terraform/assets/cluster.tf

```diff
@@ -324,7 +324,7 @@ resource "aws_rds_cluster_instance" "cluster_instances" {
   apply_immediately            = true
   auto_minor_version_upgrade   = false
   performance_insights_enabled = var.db_enable_performance_insights
-  db_parameter_group_name      = "${var.cluster_name}-db-pg"
+  db_parameter_group_name      = length(var.db_parameters) > 0 ? "${var.cluster_name}-db-pg" : ""
```

Comment:
FYI @agarciamontoro - Claudio already rebased the changes from master in this PR.

Reply:
Oh! Thanks for the heads-up :)

Reply:
Right, I actually forgot I was targeting the feature branch for a moment. @agarciamontoro feel free to merge this or let me know if you prefer it clean (no merges).

Reply:
I would prefer to avoid conflicts when we eventually merge the feature branch into master, but that may happen also if we don't maintain it up to date and change something master changed, so just 1/5 on keeping it clean.

Reply:
Thanks!